.gitignore

```diff
@@ -62,10 +62,11 @@ dist
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
 
-# AWS Docker build
+# AWS DOcker build
```

Comment:
New way of writing Docker 🤔

Reply:
Or is this a new way for 'non enterprise docker' 😂

Reply:
good spot, corrected the spelling